src/xo-app/settings/ips/index.js

```diff
@@ -97,11 +97,12 @@ class IpsCell extends BaseComponent {
             <strong>{ip}</strong>
           </Col>
           <Col mediumSize={6}>{!isEmpty(addressVifs)
-            ? map(addressVifs, vifId => {
-              const vif = vifs[vifId][0]
-              const network = networks[vif.$network][0]
-
-              return `${network.name_label} #${vif.device}`
+            ? mapPlus(addressVifs, (vifId, push) => {
+              const vif = vifs[vifId] && vifs[vifId][0]
+              const network = vif && networks[vif.$network] && networks[vif.$network][0]
+              if (network && vif) {
+                push(`${network.name_label} #${vif.device}`)
+              }
```

Comment:
IMHO, we should not hide this, but express the fact that there is an unknown VIF.